cli/generic/forms/fields.py

```diff
@@ -35,6 +37,69 @@ def _deserialize(self, value, attr, obj, **kwargs):
         return address
 
 
+class FamilyNameField(fields.Field):
+    """
+    Implements validation of the family name.
+
+    Raises a ValidationError if the family name not exist.
+
+    References:
+        - https://marshmallow.readthedocs.io/en/3.0/custom_fields.html
+    """
+
+    def _deserialize(self, value, attr, obj, **kwargs):
+        """
+        Validate data (family name) that was passed to field.
+        """
+        if value not in FAMILY_NAMES:
+            raise ValidationError(f'The following family name `{value}` is invalid.')
+
+        return value
+
+
+class TransactionIdentifierListField(fields.Field):
+    """
+    Implements validation of the list identifiers.
+
+    Raises a ValidationError if the identifier does not correspond to HEADER_SIGNATURE_REGEXP.
+
+    References:
+        - https://marshmallow.readthedocs.io/en/3.0/custom_fields.html
+    """
+
+    def _deserialize(self, value, attr, obj, **kwargs):
+        """
+        Validate data (list identifiers) that was passed to field.
+        """
+        value = [v.strip() for v in value.split(',')]
+        for id_ in value:
```

Comment:
@dmytrostriletskyi, it's a bad practice when you have very strict input rules, it bad User Experience.

I have two valid ids if I'm writing `remme transaction get-list --ids='id1, id2'` it will be work, but if i send 
`remme transaction get-list --ids='id1,   id2'` or 
`remme transaction get-list --ids='id1 , id2'` or 
`remme transaction get-list --ids='id1 ,id2'` or 
`remme transaction get-list --ids='id1, \nid2'` your system will be trow error, but id1 and id2 was valid.

Reply:
Got you. Then just do:

```
identifiers_as_array = identifiers_as_string.split(', ')

for identifier in identifiers_as_array:
    identifier = identifier.split()
    ...
```

Or something like that. Because of list comprehensions are a bit ugly for the first view and harder to avoid that just loops.

Reply:
@dmytrostriletskyi, done.